webrtc.html

```diff
@@ -8666,40 +8665,35 @@ <h3><dfn>RTCSctpTransportState</dfn> Enum</h3>
               <tr>
                 <th colspan="2">Enumeration description</th>
               </tr>
-              <tr>
-                <td><dfn data-idl><code id=
-                "idl-def-RTCSctpTransportState.new">new</code></dfn></td>
-                <td>
-                  <p>The <code><a>RTCSctpTransport</a></code>
-                  has not started negotiating yet.</p>
-                </td>
-              </tr>
               <tr>
                 <td><dfn data-idl><code id=
                 "idl-def-RTCSctpTransportState.connecting">connecting</code></dfn></td>
                 <td>
                   <p>The <code><a>RTCSctpTransport</a></code> is in the process of
-                  negotiating an association. This is the initial state when an
-                  <code><a>RTCSctpTransport</a></code> is created by applying
-                  an Answer.</p>
+                  negotiating an association. This is the initial state of the
+                  [[\SctpTransportState]] slot when an
+                  <code><a>RTCSctpTransport</a></code> is created.</p>
                 </td>
               </tr>
               <tr>
                 <td><dfn data-idl><code id=
                 "idl-def-RTCSctpTransportState.connected">connected</code></dfn></td>
                 <td>
-                  <p>The <code><a>RTCSctpTransport</a></code> has completed
-                  negotiation of an association.</p>
+                  <p>When the negotiation of an association is completed, a task is
+                  queued to update the [[\SctpTransportState]] slot to
+                  <code>"connected"</code>.</p>
                 </td>
               </tr>
               <tr>
                 <td><dfn data-idl><code id=
                 "idl-def-RTCSctpTransportState.closed">closed</code></dfn></td>
                 <td>
-                  <p>The SCTP association has been closed intentionally
+                  <p>When the SCTP association has been closed intentionally
                   (such as by closing the peer connection or applying a
                   remote description that rejects data or changes the
-                  SCTP port) or via receipt of a SHUTDOWN or ABORT chunk.</p>
+                  SCTP port) or when a SHUTDOWN or ABORT chunk is received,
```

Comment:
Grammatically, this last piece doesn't make sense any more. If you replace "or" with "." (turning "when a SHUTDOWN" into a new sentence), it makes sense; would perhaps make more sense to move the "when a SHUTDOWN" part into text about the overall operation of an SCTP association (if one exists).